CHANGELOG.md

```diff
@@ -1,3 +1,83 @@
+# v 1.12.0 (2022-01-24)
+
+## OS Changes
```

Comment:
We should add a line item for updating the host containers https://github.com/bottlerocket-os/bottlerocket/pull/2739